### ISSUE TITLE
listmp3: divide-by-zero in printMP3Headers (listmp3.c)

### ISSUE THREAD

**Issue by asarubbo** [libming/libming#52]
There is a divide-by-zero in printMP3Headers

Stacktrace:

```
ASAN:DEADLYSIGNAL
=================================================================
==29561==ERROR: AddressSanitizer: FPE on unknown address 0x0000004f19e8 (pc 0x0000004f19e8 bp 0x000000000000 sp 0x7ffdf0ab6340 T0)
    #0 0x4f19e7 in printMP3Headers /var/tmp/portage/media-libs/ming-0.4.7/work/ming-0_4_7/util/listmp3.c:172:54
    #1 0x4f1bee in main /var/tmp/portage/media-libs/ming-0.4.7/work/ming-0_4_7/util/listmp3.c:191:3
    #2 0x7f49407a361f in __libc_start_main /var/tmp/portage/sys-libs/glibc-2.22-r4/work/glibc-2.22/csu/libc-start.c:289
    #3 0x418ae8 in getenv (/usr/bin/listmp3+0x418ae8)

AddressSanitizer can not provide additional info.
SUMMARY: AddressSanitizer: FPE /var/tmp/portage/media-libs/ming-0.4.7/work/ming-0_4_7/util/listmp3.c:172:54 in printMP3Headers
==29561==ABORTING
```

Testcase:
[4.crashes.zip](https://github.com/libming/libming/files/541925/4.crashes.zip)

**Comment by fgeek** [comment]
http://www.openwall.com/lists/oss-security/2016/11/09/15

**Comment by carnil** [comment]
This is CVE-2016-9265

**Comment by strk** [comment]
Should be fixed as of Should be fixed as of b0704f80ef2ec1507043df77d6519d18cf017e16 - please reopen if it is not
